sources/001-v5/document.adoc

```diff
@@ -20,6 +20,8 @@
 :local-cache-only:
 :data-uri-image:
 
+[lutaml_ea_xmi,../xmi/plateau_all_packages_export.xmi]
```

Comment:
While this is sufficient for the current use case, what if we want to render two or more XMI files in a document? Then we need to name them just like in lutaml-express (as document attributes). 

If we do that, in declaration we define the "short name" of the XMI, location of XMI and also the base path for images. Then when we want an image, we just say for example, "image 1 in XMI 1".